sources/packages/web/src/assets/css/base.scss

```diff
@@ -79,15 +80,31 @@ html * {
   border-color: $badge-border-error !important;
 }
 
-/** Style to align records per page with arrows in data table. **/
+// Data table styles
+// Style to align records per page with arrows in data table.
```

Comment:
@ann-aot does this need to be put back using //** **// ?

Reply:
for styles, we use both.  https://github.com/bcgov/SIMS/wiki/Development---Standards,-Decisions,-and-Troubleshooting#sample-class-example

Reply:
Thanks @ann-aot !